Turner.Infrastructure.Mediator.EF/Turner.Infrastructure.Mediator.EF.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="Class1.cs" />
+  </ItemGroup>
```

Comment:
It looks like this file (`Class1.cs`) was removed from the project, but not deleted.

Reply:
No idea why that happened.  File was not in the filesystem and didn't show up in the project in VS so had to manually edit the file to remove that.

Reply:
Silly VS being VS